### PR TITLE
Fix model names

### PR DESCRIPTION
**Pull request by potatoqualitee**
Keeping `copilot-` in there breaks the extension response and sets the model to `undefined`.

### CODE REVIEW

File: api/extension-guides/language-model.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ const craftedPrompt = [
 
 ## Send the language model request
 
-Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Currently, only `copilot-gpt-3.5-turbo` and `copilot-gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
+Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Currently, only `gpt-3.5-turbo` and `gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
```
Comment (ntrogh) [inline api/extension-guides/language-model.md:74]:
```suggestion
Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Learn more about these properties in the [API reference](/api/references/vscode-api#LanguageModelChat). Currently, only `gpt-3.5-turbo` and `gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
```

Comment (ntrogh) [reply api/extension-guides/language-model.md:74]: Suggest adding a link to the API reference to get more details about the different properties.

**Review by ntrogh — CHANGES_REQUESTED** [review]
@potatoqualitee Thanks for the PR. I added a link to the API reference for more details. Otherwise, looks good to me.

File: api/extension-guides/language-model.md (hunk 1 above)
Comment (potatoqualitee) [reply api/extension-guides/language-model.md:74]: Thanks, also suggestion LGTM 😊

**Review by ntrogh — APPROVED** [review]
Thanks for the updates.

**Pull request merged**